examples/gatsby-theme-docs/gatsby-config.js

```diff
@@ -11,7 +11,6 @@ module.exports = {
     themeColor: `#8257E6`,
     basePath: `/`,
   },
-  flags: { PRESERVE_WEBPACK_CACHE: true },
```

Comment:
This is option useful for speed up compilation steps. Please let me know why did you remove it. Thanks.

Reply:
It's no longer required on Gatsby 4, as it's the default. There was a warning about it.

Reply:
@jpedroschmitz  Nice, thanks you so much for letting me know. I should have updated with Gatsby 4. ^^.